source/postprocess/particles.cc

```diff
@@ -580,7 +580,7 @@ namespace aspect
           aspect::iarchive ia (is);
 
           // Load the particle world
-          this->get_particle_world().load(is);
+          this->get_particle_world(0).load(is);
```

Comment:
This may be the only place where the non-const version of `get_particle_world` is called, and it is a bad case too. Not for this PR, but in the future I want to look into moving the functionality for serializing/deserializing the world into Simulator, since you may have a particle world, but no particle postprocessor.